firestore/main/index.js

```diff
@@ -587,7 +587,7 @@ async function queryAndFilter(db) {
   const citiesRef = db.collection('cities');
 
   // Create a query against the collection
-  const allCapitalsRes = await citiesRef.where('capital', '==', true).get();
+  const allCapitalsRes = await citiesRef.where('capital', '==', true);
```

Comment:
We can get rid of the `await` as well:
```suggestion
  const allCapitalsRes = citiesRef.where('capital', '==', true);
```